### DISCUSSION TITLE
[HKSV] Anyway to disable re-encoding when recording to HKSV?

### DISCUSSION THREAD

**Discussion by jimsmt** [koush/scrypted#677] [General]
Video source is 4K bitrate 8Mbps, home hub(Apple TV 4K) will re-encode it to 1Mbps when recording to HKSV, image quality is much worse than the original. I tried different bitrate for the source video, from 1Mbps to 16Mbps, HKSV video will always be compressed to about 1Mbps, resolution stay 4K however

Tried setting HomePod2/mini as home hub, which use Apple Watch’s SoC and don’t have ability to re-encode a 4K stream in realtime. I thought it may just forward the original 8Mbps stream directly to HKSV server without re-encoding, but actually it just refuses to process it. The highest resolution I succeed with HKSV recording on HomePod2 hub is 1920x1440, while with Apple TV 4K, I can record a 4000x3000 stream, although still re-encoded to 1Mbps

**Comment by koush** [comment]
no
